examples/rust/standalone/qdrant-example/src/main.rs

```diff
@@ -0,0 +1,190 @@
+    // Note that the encrypted result will not be deterministic because of a different IV each run.
+    dbg!(&search_result);
+    // &search_result = SearchResponse {
+    //     result: [
+    //         ScoredPoint {
+    //             id: Some(
+    //                 PointId {
+    //                     point_id_options: Some(
+    //                         Num(
+    //                             0,
+    //                         ),
+    //                     ),
+    //                 },
+    //             ),
+    //             payload: {
+    //                 "_encrypted_document": Value {
+    //                     kind: Some(
+    //                         StringValue(
+    //                             "AAAAAYIAAG8KJAograI69h2cKUS97PbwJEgbjYbZbQLrQD2I9BHTq2a38X8QARJHEkUaQwoMQ54URzJDiUNW7G5JEjCPhU1WpnaD8cMkmr+zRjbya+QHvAh5lNwIucUgEiOFCxmwixy0t3tAN6dyhf+OPx4aATGHaBVZrqZ6GwALR9HOoc2sOMyvQXyEMwZLRWfUcO25pKkIGuNLf02zAqijyDH2YXJ5TJsAAxsUcDXBUkY9qEJui8qIAQQlLA",
+    //                         ),
+    //                     ),
+    //                 },
+    //             },
+    //             score: 0.9999996,
+    //             version: 0,
+    //             vectors: None,
+    //             shard_key: None,
+    //         },
+    //     ],
+    //     time: 0.000914,
+    // }
+
+    let found_point = search_result.result.into_iter().next().unwrap();
+    let mut payload = found_point.payload;
+    let encrypted_payload = payload.remove(ENCRYPTED_DOC_KEY).unwrap();
+    // Since the payload has the encrypted, get the encrypted bytes.
+    let encrypted_bytes = base64_engine
+        .decode(encrypted_payload.as_str().unwrap())
+        .unwrap();
+    let decrypted_document = alloy_client
+        .standard_attached()
+        .decrypt(
+            ironcore_alloy::standard_attached::EncryptedAttachedDocument(encrypted_bytes),
+            &alloy_metadata,
+        )
+        .await?;
+    println!(
+        "Decrypted document: {}",
+        std::str::from_utf8(&decrypted_document)?
+    );
+
+    Ok(())
+}
+
+fn create_plaintext_vector(v: Vec<f32>) -> PlaintextVector {
+    PlaintextVector {
+        plaintext_vector: v,
+        secret_path: SecretPath("".to_string()),
+        derivation_path: DerivationPath("".to_string()),
+    }
+}
+
+fn create_alloy_client() -> Result<std::sync::Arc<Standalone>> {
+    // An example key, obviously this would come from a secure location in a real situation.
+    let standalone_secret = "f8cba200fb44b891d6a389858ae699d57b7ff48572cdcb9e5ed1d4364bf531b8";
+    let config = StandaloneConfiguration::new(
+        StandardSecrets::new(
+            Some(1),
+            vec![StandaloneSecret::new(
+                1,
+                Secret::new(standalone_secret.as_bytes().to_vec()).unwrap(),
+            )],
+        )
+        .unwrap(),
+        HashMap::new(),
+        HashMap::from([(
+            SecretPath("".to_string()),
+            VectorSecret::new(
+                0.1,
```

Comment:
Our other examples have been using an approximation factor of ~2-2.5. I would increase this to make it more "real"